src/upf/src/CMakeLists.txt

```diff
@@ -61,7 +73,7 @@ target_include_directories(upf
 
 target_link_libraries(upf 
   PRIVATE
+    upf_lib
     utl
-    dbSta
-    OpenSTA
+    odb
```

Comment:
upf_lib already depends on odb

Reply:
The SWIG also depends on odb headers which are not included by default. You need to call out odb explicitly.

Reply:
Since upf.h includes db.h, odb should be a public dependency of upf_lib not a private one and this would be unnecessary.

Reply:
Hmm, I'm not a C++ guru, but at least at Google it would still be required to declare the dependency because a public header of rcx also imports odb headers.

We're not allowed to depend on the headers of our transitive dependencies.

Reply:
@cdleary do I have the above right?

Reply:
I don't agree in this case but don't care enough to debate it.